azext_iot/central/commands_device.py

```diff
@@ -0,0 +1,51 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# Dev note - think of this as a controller
+
+from knack.util import CLIError
+from .providers import CentralDeviceProvider
```

Comment:
For this project please use fully qualifier imports in the production/packaged code - test modules can be more flexible though.